src/components/pages/Profile/Header/Stats.tsx

```diff
@@ -32,61 +32,64 @@ export default function Stats({
 }: {
   accountId: string
 }): ReactElement {
-  const { chainIds } = useUserPreferences()
+  /**
+   * Comments in this file should stay to easily reintroduce stats again when necessary
+   */
+  // const { chainIds } = useUserPreferences()
   const { poolShares, assets, assetsTotal, sales } = useProfile()
 
-  const [publisherLiquidity, setPublisherLiquidity] = useState<UserLiquidity>()
-  const [totalLiquidity, setTotalLiquidity] = useState(0)
+  // const [publisherLiquidity, setPublisherLiquidity] = useState<UserLiquidity>()
+  // const [totalLiquidity, setTotalLiquidity] = useState(0)
 
-  useEffect(() => {
-    if (!accountId || chainIds.length === 0) {
-      setPublisherLiquidity({ price: '0', oceanBalance: '0' })
-      setTotalLiquidity(0)
-    }
-  }, [accountId, chainIds])
+  // useEffect(() => {
+  //   if (!accountId || chainIds.length === 0) {
+  //     setPublisherLiquidity({ price: '0', oceanBalance: '0' })
+  //     setTotalLiquidity(0)
+  //   }
+  // }, [accountId, chainIds])
```

Comment:
question(blocking): Why are we keeping commented code here?

Reply:
The idea was to keep comments in this file to easily reintroduce stats again when necessary. If we decide we won't need those stats at all for this market version we could remove the code completely - I felt it might be of use at some point in time.